flink-runtime/src/main/java/org/apache/flink/runtime/jobgraph/tasks/CheckpointCoordinatorConfiguration.java

```diff
@@ -63,11 +65,13 @@ public CheckpointCoordinatorConfiguration(
 			int maxConcurrentCheckpoints,
 			CheckpointRetentionPolicy checkpointRetentionPolicy,
 			boolean isExactlyOnce,
-			boolean isPerfetCheckpointForRecovery) {
+			boolean isPerfetCheckpointForRecovery,
+			int tolerableCpFailureNumber) {
 
 		// sanity checks
-		if (checkpointInterval < 1 || checkpointTimeout < 1 ||
-			minPauseBetweenCheckpoints < 0 || maxConcurrentCheckpoints < 1) {
+		if (checkpointInterval < 10 || checkpointTimeout < 10 ||
```

Comment:
Is there any good reason why the constant here is changed from 1 to 10 in both cases?

Reply:
Yes, I have a good reason. `CheckpointConfig#setCheckpointInterval` and `CheckpointConfig#setCheckpointTimeout` consider the illegal value is (`< 10`). So we'd better have the same criterion.